share/source.F90

```diff
@@ -125,21 +130,28 @@ subroutine source
   use mcnp_random, only: rang
   use pblcom, only: xxx, yyy, zzz, erg, tme, wgt, icl, ipt, jsu
   use mcnp_debug
+  use varcom, only: nps, npp
 
   implicit none
 
   logical, save :: first_run = .true.
   real(dknd), dimension(6) :: rands
   integer :: icl_tmp ! temporary cell index variable
-  integer :: find_cell
+  !integer :: find_cell
```

Comment:
Delete this line instead of comment

Reply:
Changed in https://github.com/pyne/pyne/pull/1159/commits/e23c72e7bf794e3521cdfedc9dd4ea857456f191.